File: Python/Problem-16.py
# Write a Python Program to Print The Fibonacci Series 

a = 0
b = 1
# s = a +b
number = int(input("Enter Any Number: "))

if number == 1:
  print(a)
else:
  print(a)
  print(b)
  for i in range(2,number):
    c = a+b
    a = b
    b = c
    # s = s + c
    print(c)
  print(f"The sum of Fib is {s}")

File: JavaScript/Problem-16.js
// # Write a Python Program to Print The Fibonacci Series
const prompt = require("prompt-sync")();
var a = 0;
var b = 1;
var c = 0;
var sum = a + b
var number = parseInt(prompt("Enter Any Number: "));

if (number == 1) {
  console.log(a);
} else {
  console.log(a);
  console.log(b);
  for (let i = 2; i < number; i++) {
    c = a + b;
    a = b;
    b = c;
    sum = sum + c
    console.log(c);
  }
  console.log(`Sum Of Fib is ${sum}`)
}
